lib/pg.utils.js

```diff
@@ -1,5 +1,22 @@
 'use strict';
 
+const CATEGORY_TYPES = {
+  Registry: 'Global',
+  Dictionary: 'Global',
+  System: 'Global',
+  History: 'Global',
+  Local: 'Local',
+  Log: 'Ignore',
+  View: 'Ignore',
+  Memory: 'Ignore',
+};
+
+const getCategoryType = type => CATEGORY_TYPES[type] || 'Local';
+
+const isGlobal = type => getCategoryType(type) === 'Global';
+const isIgnored = type => getCategoryType(type) === 'Ignore';
+const isLocal = type => getCategoryType(type) === 'Local';
```

Comment:
IMO It's not clear from the name that this relates to categories, maybe `isLocalCategory` and such?

Also, it doesn't look like this is specific to pg, so perhaps move it to the other file?